index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <title>Bootstrap demo</title>
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.2/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+      integrity="sha384-T3c6CoIi6uLrA9TneNEoa7RxnatzjcDSCmG1MXxSR1GAsXEV/Dwwykc2MPK8M2HN"
+      crossorigin="anonymous"
+    />
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <div class="container my-5">
+      <h1>Hello, world!</h1>
+      <div class="col-lg-8 px-0">
+        <hr class="col-1 my-4" />
+
+        <a href="git@github.com:madejsoon/hello-world.git" class="btn btn-primary"
```

Comment:
It should be `https://github.com/madejsoon/hello-world` not the one you used.

Reply:
look now

Reply:
LGTM